PaddleHQ/transaction-search
# TOOLS-263 Fix casing of request and response keys

### Ticket(s):
Link to ticket:
https://paddle.atlassian.net/browse/TOOLS-263

### Checklist

### What has been done:
snake_case keys have been replaced with lowerCamelCase keys.

### How to test:
Make a few requests on your environment and confirm that the response always contains keys with the correct case.
